Mobile/iOS/Templates/Release_3.5.0/xxx-text-editor.md

```diff
@@ -0,0 +1,32 @@
+###  Text editor 
+
+#### Pr: https://github.com/owncloud/ios/pull/XXX 
+
+
+---
+
+ 
+| TestID | Test Case | Steps | Expected Result | Result | Related Comment |
+| :----: | :-------- | :---- | :-------------- | :----: | :-------------- |
+|**View**||||||
+| 1 | Text preview |  1. Open a text file file (txt, rtf, css, py, xml, js)<br>2. Change orientation  | 1. On top right corner appears the option "Edit".<br> 2. Option is correctly displayed |  |  |
+| 2 | No-text preview |  Open a no-txt file (doc, xls, pdf...)  | 1. On top right corner does not appear the option "Edit". |  |  |
+| 3 | Edit view |  1. Open a txt file and tap on "Edit" <br>2. Change orientation  | 1. Buttons cancel and done are shown. The name of the file is shown on the title. Tab bar is not shown<br> 2. View is correctly displayed |  |  |
+|**Text edition**||||||
+| 4 | Edit text |  1. Open edit view<br>2. Edit the content<br>3. Tap on "Done" | Content is uploaded (check in server) and file view is shown with new modification date |  |  |
+| 5 | Edit without changes |  1. Open edit view<br>2. Do not edit the content<br>3. Tap on "Done" | Content is not uploaded and file view is shown with old modification date |  |  |
+| 6 | Cancel edition |  1. Open edit view<br>2. Edit the content<br>3. Tap on "Cancel" | Content is not uploaded (check in server) and file view is shown with old modification date |  |  |
+| 7 | Cancel without changes |  1. Open edit view<br>2. Do not edit the content<br>3. Tap on "Cancel" | Content is not uploaded (check in server) and file view is shown with old modification date |  |  |
+| 8 | Consecutive changes |  Edit a file several times | Content is uploaded (check in server) and file view is shown with old modification date. Uploads view shows all the times that the file was chamged |  |  |
+|**External actions**||||||
+| 9 | Deleted file |  1. Open edit view<br>2. In server, delete the file<br>3. Tap on "Done" | File does not appear in file view |  |  |
```

Comment:
Expected behaviour: File is upload correctly
All this section should work as updated files works when you edit them from document provider